nemo/collections/nlp/modules/common/megatron/adapters/parallel_adapters.py

```diff
@@ -112,7 +110,6 @@ def __init__(
         row_init_method: str = 'zero',  # TODO: (@adithyare) should rename this to output_init_method to be more precise.
         gather_output: bool = True,
         dropout: float = 0.0,
-        adapter_strategy: adapter_mixin_strategies.ResidualAddAdapterStrategyConfig = None,
```

Comment:
I think in some place, the adapter engine does try to call the module with this strategy - hopefully removing it does not break anything. 
An alternative is to simply replace the default value.

Reply:
Calls to ParallelLinearAdapter don't explicitly pass a strategy arg so nothing breaks. I want the strategy to be explicitly set rather than defaults because its easier to catch bugs this way.

Reply:
Makes sense